controllers/clusterversion_controller.go

```diff
@@ -213,6 +213,9 @@ func (c *ClusterVersionReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		if err := c.own(c.cephFSDeployment); err != nil {
 			return err
 		}
+		if c.cephFSDeployment.Labels == nil {
+			c.cephFSDeployment.Labels = make(map[string]string)
+		}
```

Comment:
move this code to SetCephFSDeploymentDesiredState function. and also please take care of the same in other places also

Reply:
ya, initially I thought of doing it that way but wasn't sure a set function should create this or not.

Reply:
done.